flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/JobMasterTester.java

```diff
@@ -239,21 +235,23 @@ private TaskExecutorGateway createTaskExecutorGateway() {
 
     private CompletableFuture<Acknowledge> onSubmitTaskConsumer(
             TaskDeploymentDescriptor taskDeploymentDescriptor, JobMasterId jobMasterId) {
-        return executionGraphInfoFuture.thenCompose(
```

Comment:
Can't we remove `executionGraphInfoFuture`?

Reply:
It's already removed. The intuition behind this is, that adaptive scheduler only provides a "sparse" execution graph during first states in it's lifecycle.